birdhouse/components/stac/docker-compose-extra.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  stac:
+    container_name: stac
+    image: ghcr.io/crim-ca/stac-app:main
+    depends_on:
+      - stac-db
+    environment:
+      - POSTGRES_USER=${STAC_POSTGRES_USER}
+      - POSTGRES_PASS=${STAC_POSTGRES_PASSWORD}
+      - POSTGRES_DBNAME=postgis
```

Comment:
Just curious, is `postgis` hardcoded somewhere?  Why not call the DB simply `stac`?